README.md

```diff
@@ -0,0 +1,52 @@
+# SKonverto
+SKonverto is a tool to transform Lambdapi proofs containing Skolem symbols into 
+proofs without these symbols.
+
+## Installation
+
+### Dependencies
+    - dune
+    - lambdapi (https://github.com/Deducteam/lambdapi)
+    - OCaml >= 4.07.0
+### Compilation
+```bash
+    git clone https://github.com/elhaddadyacine/SKonverto
+    cd SKonverto
+    make
+    make install
+```
+
+### Use
+```bash
+    skonverto path_to_the_lambdapi_file.lp
+```
+
+### Example
+```
+require open logic.fol logic.nd logic.nd_eps logic.zen; // files available in 
+constant symbol a : κ;
+constant symbol p : κ → κ → Prop;
+constant symbol s : κ → κ;
+constant symbol f : κ → κ;
+symbol axiom_A :  ϵ (`∀ X, `∃ Y, (p X (s  Y) ));

```

Comment:
thanks to add a comment before: `// axiom that is skolemized`.
It would be good to give hereafter the skolemized version. Hence, the declaration of f should come here too.